appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js

```diff
@@ -13,27 +13,51 @@ const appealDataWithoutStartDate = {
 };
 
 describe('start-case', () => {
-	beforeEach(() => {
-		nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
-	});
 	afterEach(teardown);
 
 	describe('GET /start-case/add', () => {
 		it('"Start case" page should contain "Confirming will activate the timetable on"', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
 			const response = await request.get(`${baseUrl}/1/start-case/add`);
 			expect(response.text).toContain('Confirming will activate the timetable on');
 		});
 
 		it('should redirect to the confirmation page', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
 			nock('http://test/').post(`/appeals/1/appeal-timetables`).reply(200, {});
 			const response = await request.post(`${baseUrl}/1/start-case/add`).send({});
 			expect(response.statusCode).toBe(302);
 		});
 	});
 
-	describe('GET /start-case/confirmation', () => {
+	describe('GET /start-case/add/confirmation', () => {
 		it('should render "Case started" confirmation page', async () => {
-			const response = await request.get(`${baseUrl}/1/start-case/confirmation`);
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
+			const response = await request.get(`${baseUrl}/1/start-case/add/confirmation`);
+			const element = parseHtml(response.text);
+			expect(element.innerHTML).toMatchSnapshot();
```

Comment:
Just a suggestion, you could add some quick checks for content you expect to be present in the snapshot, to help avoid regressions - there's some examples in `appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/__tests__/manage-linked-appeals.test.js` - there's a tech debt ticket (https://pins-ds.atlassian.net/browse/BOAT-1407) which covers updating existing instances where this is missing, but I think we should probably try to add these for any new tests like this one

Reply:
done